src/ui/app/jsx/repair-schedule-form.jsx

```diff
@@ -110,7 +110,7 @@ const RepairScheduleForm = CreateReactClass({
     const segmentsPerNodeInvalid = !state.segments ||
       state.segments == undefined ||
       state.segments == null ||
-      state.segments < 1 ||
+      state.segments < 0 ||
```

Comment:
The segment count per node is considered invalid by default because it's initialized as `''` instead of `0` here: https://github.com/thelastpickle/cassandra-reaper/pull/1119/commits/ec5580723fa0dbf813547c1a04ca391856222830#diff-166c0877429b0e6dad5328286ab5af08f46ff06c0ee1a048483f52ffa5dd9d3eR43

This results in the `Save` button being disabled unless the value is changed.
Could you make it so that 0 is the default value and the `Save` button is enabled when editing without making a change?

Reply:
It's definitely true that the Save button is disabled until a value is changed, but not because of the validity of the value.  The validation check is only triggered on changes to the form, so that you can't save an non-dirty form so to speak.  It will only let you save a form that has been changed -- although it's not a deep inspection that compares each value over and over again, just for the record.

But, that aside, I do notice that I have the validation wrong, I shouldn't be checking !state.segments, because it's a number and that could screw up, I will fix that now.

Reply:
I also failed to check the intensity value there, I've added that as well in the new commit.